manifest.json

```diff
@@ -291,7 +285,7 @@
         {
           "sharedTags": {
             "1.0-2.0": {},
-            "1.0-2.0-2017-10": {
+            "1.0-2.0-2017-11": {
```

Comment:
Curious if there is a consumer of these and why they aren't documented.

Reply:
Was never clear on the justification either. cc @glennc - do we really care about this anymore?